test/shared_utilities/variable_types.jl

```diff
@@ -85,32 +92,22 @@ end
     ClimaLSM.auxiliary_vars(m::Model) = (:d, :e)
 
     ClimaLSM.prognostic_types(m::Model{FT}) where {FT} = (FT, SVector{2, FT})
-
     ClimaLSM.auxiliary_types(m::Model{FT}) where {FT} = (FT, SVector{2, FT})
 
+    ClimaLSM.prognostic_domain_names(m::Model) = (:surface, :surface)
+    ClimaLSM.auxiliary_domain_names(m::Model) = (:surface, :subsurface)
+
     FT = Float64
     zmin = FT(1.0)
     zmax = FT(2.0)
     zlim = (zmin, zmax)
     nelements = 5
 
     column = Column(; zlim = zlim, nelements = nelements)
-    point = Point(; z_sfc = zlim[1])
-
-    for domain in [column, point]
-
-        m = Model{FT, typeof(domain)}(domain)
-        Y, p, coords = initialize(m)
-        if typeof(domain) <: Column
-            @test parent(Y.foo.a) == zeros(FT, 5, 1)
-            @test parent(Y.foo.b) == zeros(FT, 5, 2)
-            @test parent(p.foo.d) == zeros(FT, 5, 1)
-            @test parent(p.foo.e) == zeros(FT, 5, 2)
-        elseif typeof(domain) <: Point
-            @test Y.foo.a[] == 0.0
-            @test Y.foo.b[] == zero(SVector{2, FT})
-            @test p.foo.d[] == 0.0
-            @test p.foo.e[] == zero(SVector{2, FT})
-        end
-    end
+    m = Model{FT, typeof(column)}(column)
+    Y, p, coords = initialize(m)
+    @test parent(Y.foo.a) == zeros(FT, 1)
+    @test parent(Y.foo.b) == zeros(FT, 2, 1)[:]
```

Comment:
why does this line have `[:]`?

Reply:
fixed this! no longer needs it - we should just compare to zeros(FT,2), not make a 2x1 object and then flatten it.